src/backend/main.ts

```diff
@@ -252,7 +252,7 @@ async function initializeWindow(): Promise<BrowserWindow> {
   } else {
     Menu.setApplicationMenu(null)
     mainWindow.loadFile(join(publicDir, 'index.html'))
-    if (globalConf.checkForUpdatesOnStartup && !isLinux) {
+    if (globalConf.checkForUpdatesOnStartup) {
```

Comment:
Since the update message now has a case for auto-update not being supported, we want to always check for updates here